implementations/rust/ockam/ockam_command/tests/bats/command-reference.bats

```diff
@@ -0,0 +1,344 @@
+#!/bin/bash
+
+# ===== SETUP
+
+setup() {
+  load load/base.bash
+  load load/orchestrator.bash
+  load_bats_ext
+  setup_home_dir
+  skip_if_orchestrator_tests_not_enabled
+  copy_local_orchestrator_data
+}
+
+teardown() {
+  teardown_home_dir
+}
+
+# ===== TESTS https://docs.ockam.io/reference/command/nodes
+@test "nodes" {
+  run "$OCKAM" node create
```

Comment:
I was wondering if we could write a `run_success` function which does `run xxx; assert_success` in one line. What do you think?

Reply:
AFAIK, we could replace the `run xxx; assert_success` pattern by just `xxx` because bats scripts are configured to exit on error by default. I read this recently, I'll try to post the link to the source if I remember where.

Reply:
Although, having the `run_success` you mentioned will make things more explicit and easier to understand.